platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/FilterInjector.java

```diff
@@ -146,24 +146,26 @@ private void injectFilter(ServletContext context, Bundle refBundle) {
           "Failed trying to set the cookie config path to /. This can usually be ignored", e);
     }
 
-    try {
+    for (Filter filter : filterList) {
+      try {
 
-      FilterRegistration filterReg =
-          context.addFilter(SECURITY_JAVA_SUBJECT_FILTER, securityJavaSubjectFilter);
+        FilterRegistration filterReg = context.addFilter(SECURITY_JAVA_SUBJECT_FILTER, filter);
 
-      if (filterReg == null) {
-        filterReg = context.getFilterRegistration(SECURITY_JAVA_SUBJECT_FILTER);
-      } else {
-        ((FilterRegistration.Dynamic) filterReg).setAsyncSupported(true);
-      }
+        if (filterReg == null) {
+          filterReg = context.getFilterRegistration(SECURITY_JAVA_SUBJECT_FILTER);
```

Comment:
❗️ I'm guessing this could cause bad side effects, using the same "name" for all filter registrations. Does it make more sense to use the classname of the filter?

Reply:
Yeah that should be fine. I just forgot to remove this.